cloud/datasync/src/SyncServer.ts

```diff
@@ -4,6 +4,16 @@ import {setupGlobalHandlers, setupHandlers} from './crud-handlers/HandlerMapper'
 import SyncDataSetOptions from './options/SyncDatasetOptions';
 import SyncOptions from './options/SyncGlobalOptions';
 import SyncApi from './SyncApi';
+import {BunyanLogger,LOG_LEVEL} from '../../logger-cloud/src/index';
+//import {BunyanLogger,LOG_LEVEL} from '@raincatcher/logger-cloud'; this don't work
+
+// constructor for loggers
+const logD = new BunyanLogger(LOG_LEVEL.DEBUG);
```

Comment:
+1, multiple instances should be for prefixing messages and so on, see https://github.com/trentm/node-bunyan#logchild and the debug function: https://github.com/visionmedia/debug